controller/workitem_blackbox_test.go

```diff
@@ -1659,6 +1660,170 @@ func (s *WorkItem2Suite) TestWI2ListByStateFilterOKModifiedUsingIfNoneMatchIfMod
 	require.Empty(s.T(), toBeFound, "failed to find these work items in list: %+v", toBeFound)
 }
 
+func (s *WorkItem2Suite) TestListOrder() {
+	s.T().Run("list with explicit order", func(t *testing.T) {
+		fxt := tf.NewTestFixture(t, s.DB,
+			tf.Iterations(2),
```

Comment:
Why are you creating two iterations here?

Reply:
Fixed in 902561db05682910b875a49091a397a1f86ef227